api/TimeAddressableMediaStore.yaml

```diff
@@ -1951,11 +1951,10 @@ paths:
         A service instance MAY support Media Objects that are held in external storage in another TAMS or other media storage system.
         The Flow Segment may in that case require the `get_urls` property to provide the information needed by clients to access the Media Object.
 
-        Clients MAY modify Flow Segments, but this should only be done in exceptional circumstances to correct metadata as such operations will likely break the idempotency of Segments.
-        Properties of Media Objects, such as `get_urls`, SHOULD be modified via the [`/objects`](#/operations/GET_objects) endpoints.
-        They SHOULD NOT be modified at this endpoint, and TAMS instences SHOULD reject such requests with a `400` error.
+        The list of instances of an object (and associated `get_urls` entries) can be modified via the [`/objects`](#/operations/GET_objects) endpoints, which provides a mechanism to register new instances of an object.
 
-        If a client needs to modify a Flow Segment, e.g. to correct metadata such as the `key_frame_count` or add additional URLs to `get_urls`, then the client SHOULD first delete the existing Segment and then write a new one.
+        Clients MAY modify Flow Segments, but this should only be done in exceptional circumstances to correct metadata such as `key_frame_count`, as such operations will likely break the idempotency of Segments.
+        If a client needs to modify a Flow Segment, then the client SHOULD first delete the existing Segment and then write a new one.
```

Comment:
Should we also add a note, like those we have elsewhere, warning that setting `key_frame_count` to a value that conflicts with the Object SHOULD be rejected if that object is in use elsewhere? i.e. don't bring references into conflict with each other.